src/compiler/transformers/declarations.ts

```diff
@@ -684,7 +699,7 @@ export function transformDeclarations(context: TransformationContext) {
         return ret;
     }
 
-    function filterBindingPatternInitializersAndRenamings(name: BindingName) {
+    function filterBindingPatternInitializers(name: BindingName) {

```

Comment:
This function still modifies renamings so I would probably leave the name as-is.